go.mod

```diff
@@ -10,14 +10,16 @@ require (
 	k8s.io/api v0.25.2
 	k8s.io/apimachinery v0.25.2
 	k8s.io/client-go v0.25.2
+	knative.dev/control-protocol v0.0.0-20221026172758-74766ce15fd4
 	knative.dev/hack v0.0.0-20221104013908-8f3c7050408b
 	knative.dev/networking v0.0.0-20221104155004-0d4e93709170
 	knative.dev/pkg v0.0.0-20221107171117-0243d641354d
+	knative.dev/serving v0.34.2
```

Comment:
I use something from `knative.dev/serving/pkg/networking`  here: https://github.com/KauzClay/net-contour/blob/b869980ff464728086ed56447958bcfd4ec84479/pkg/reconciler/contour/resources/httpproxy.go#L194

Maybe I'm doing something wrong, but when I try to import it, it brings in all of serving, not just the networking pkg

Reply:
yeah. that was there when I started, and I figured it lived in serving since that is also where the code that actually populates the secrets with data lives

Reply:
yeah, I didn't realize it was in the networking package, not the serving package. My bad

Reply:
hmmm this is kind of hard to keep track of.  

`serving/pkg/networking/config/constants.go` defines ServingCertName as "serving-certs" here: https://github.com/knative/serving/blob/main/pkg/networking/constants.go#L57

But then the link you provided defines ServingInternalCertName as "knative-serving-certs". 

In my use case, I want the value "serving-certs" because the[serving code uses that value to append to the name of the internal serving certs created in the namespace where the ksvc is](https://github.com/knative/serving/blob/ce2a9f605c5390b67d8d215513529033d3b5217d/pkg/reconciler/revision/reconcile_resources.go#L205), and that is the cert Contour needs to be configured to use.

Reply:
Yup